scrypto/src/constants.rs

```diff
@@ -25,3 +25,8 @@ pub const RADIX_TOKEN: ResourceAddress = ResourceAddress([
 pub const ECDSA_TOKEN: ResourceAddress = ResourceAddress([
     0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 5,
 ]);
+
+/// The system token which only validators have access to
```

Comment:
It's not just validators - it's more the system executor, or I think we called them "system transactions", I believe?